### PR TITLE
Add /engage index

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- created a very simple /enage index page for SEO

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page has a very simple list of all engage pages

### CODE REVIEW

File: templates/engage/index.html [hunk 1]
```diff
@@ -0,0 +1,1026 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}Ubuntu resources{% endblock %}
```
Comment (caldav) [inline templates/engage/index.html]:
```suggestion
{% block title %}Ubuntu case studies, whitepapers and webinars{% endblock %}
```

**Comment by anasereijo** [comment]
@pmahnke this is looking great, just wondering if we need a title?

**Comment by pmahnke** [comment]
@anasereijo 	like an h1?

**Comment by anasereijo** [comment]
@pmahnke yeah just to give the user a bit of context if they land on this page. Not sure what to call it though... Engage doesn't sound good, does it?

**Comment by pmahnke** [comment]
@caldav 	you need to give the code itself a +1

**Pull request merged**